src/feedback.py

```diff
@@ -0,0 +1,23 @@
+from src.graph_model import SecurityEvent, Feedback
+from src.graph_traversel import Traversel
+from src.gremlin import GREMLIN
+from src.types import FeedBackType
+
+def _ingest_feedback(payload):
+    g = Traversel()
+    # create SecurityEvent obj with only url as property, which is enough
+    # to find the existing node to add feedback
+    security_event = SecurityEvent(url=payload['url'])
```

Comment:
may be we can move all payload properties to some constant file and read it everywhere, that way if any change require its easy to change only one place.

Reply:
I don't think it a better pattern to use const to refer JSON payload.